libs/domains/services/feature/src/lib/service-details/service-details.tsx

```diff
@@ -0,0 +1,221 @@
+import { type ComponentPropsWithoutRef } from 'react'
+import { Link } from 'react-router-dom'
+import { match } from 'ts-pattern'
+import { IconEnum, ServiceTypeEnum } from '@qovery/shared/enums'
+import {
+  Badge,
+  DescriptionDetails as Dd,
+  DescriptionListRoot as Dl,
+  DescriptionTerm as Dt,
+  Icon,
+  IconAwesomeEnum,
+  Section,
+  Tooltip,
+  Truncate,
+} from '@qovery/shared/ui'
+import { dateFullFormat, timeAgo } from '@qovery/shared/util-dates'
+import { formatBytes, formatCronExpression, twMerge } from '@qovery/shared/util-js'
+import { useService } from '../hooks/use-service/use-service'
+import { LastCommit } from '../last-commit/last-commit'
+import { ServiceDetailsSkeleton } from './service-details-skeleton'
+
+export interface ServiceDetailsProps extends ComponentPropsWithoutRef<'div'> {
+  environmentId: string
+  serviceId: string
+}
+
+export function ServiceDetails({ className, environmentId, serviceId, ...props }: ServiceDetailsProps) {
+  const { data: service } = useService({
+    environmentId,
+    serviceId,
+  })
+
+  if (!service) {
+    return <ServiceDetailsSkeleton className={className} {...props} />
+  }
+
+  const containerImage = match(service)
+    .with({ serviceType: ServiceTypeEnum.JOB }, ({ source }) => source?.image)
+    .with({ serviceType: ServiceTypeEnum.CONTAINER }, ({ image_name, tag }) => ({ image_name, tag }))
+    .otherwise(() => undefined)
+
+  const databaseSource = match(service)
+    .with({ serviceType: ServiceTypeEnum.DATABASE }, ({ accessibility, mode, type, version }) => ({
+      accessibility,
+      mode,
+      type,
+      version,
+    }))
+    .otherwise(() => undefined)
+
+  const ResourceUnit = ({ value, description }: { value?: string | number | null; description: string }) => {
+    if (value === undefined || value === null) {
+      return
+    }
+    return (
+      <div className="grid grid-row-2">
+        <span className="text-lg text-neutral-400 font-semibold">{value}</span>
```

Comment:
Should be 16px

```suggestion
        <span className="text-base text-neutral-400 font-semibold">{value}</span>
```

Reply:
https://github.com/Qovery/console/pull/865/commits/afcf4d0338229cdfeff146f720bbdb191ee9393d